### PR TITLE
replace elastica search and list by class constant

### PR DESCRIPTION
**Pull request by djamnazi**
[OO-MISC] replace strings elastica_search and elastica_list by class constant

### CODE REVIEW

File: ElasticaAdmin/BlockParameter/ElasticSearchStrategy.php [hunk 1]
```diff
@@ -10,6 +10,8 @@
  */
 class ElasticSearchStrategy implements BlockParameterInterface
 {
+    const NAME = 'elastica_search';
```
Comment (ngilain) [inline ElasticaAdmin/BlockParameter/ElasticSearchStrategy.php:13]: If you want to be coherent with what you've done on ElasticaFront/DisplayBlock/ElasticaSearchStrategy.php, you should alter the getName method too (l38)

File: ElasticaAdmin/BlockParameter/ElasticSearchStrategy.php [hunk 2]
```diff
@@ -17,7 +19,7 @@ class ElasticSearchStrategy implements BlockParameterInterface
      */
     public function support(BlockInterface $block)
     {
-        return in_array($block->getComponent(), array ('elastica_search', 'elastica_list'));
+        return in_array($block->getComponent(), array (self::NAME, 'elastica_list'));
```
Comment (ngilain) [inline ElasticaAdmin/BlockParameter/ElasticSearchStrategy.php:22]: What about the 'elastica_list' string ? I think there must be somewhere an ElasticaListStrategy to update too

File: ElasticaAdmin/DisplayBlock/ElasticaSearchStrategy.php [hunk 3]
```diff
@@ -11,6 +11,8 @@
  */
 class ElasticaSearchStrategy extends AbstractStrategy
 {
+    const NAME = 'elastica_search';
```
Comment (ngilain) [inline ElasticaAdmin/DisplayBlock/ElasticaSearchStrategy.php:14]: Maybe you can alter getName here too

File: ElasticaAdmin/BlockParameter/ElasticSearchStrategy.php (hunk 1 above)
Comment (djamnazi) [inline ElasticaAdmin/BlockParameter/ElasticSearchStrategy.php:13]: I actually did not alter the getName to remain coherent as it is done in others classes NicolasT is agree with that.

**Pull request closed without merging**